stages/org.osbuild.machine-id

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+"""
+Deal with /etc/machine-id
+
+Explicitly define the state to /etc/machine-id.
+"""
+
+import pathlib
+import sys
+
+import osbuild.api
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["machine-id"],
+"properties": {
+  "machine-id": {
+    "enum": ["first-boot", "empty", "preserve"],
```

Comment:
Super nit-pick: I like that you called the first member of the enum `first-boot` as it describes the semantics rather than what you put in the file. Would be nice to also name the second one similarly, so rather than `empty` name it after what the intended semantics is (`preserve` I think is correct as is).

Reply:
I was really unsure what would be a good value (hence not suggesting anything ;) ). I think `generate` might make sense, but really what it does (if I got the subtleties right) is `subsequent-boot` (obviously can't call it that...). I think `first-boot` would also generate one, right? It would just change the behaviour after doing so?

Reply:
Hm, what about calling the property `first-boot`, rather than `machine-id` and the enum `yes`, `no`, `preserve` or something like that?

Reply:
I like this as well. I think it reads nicely:
```
"properties": {
  "first-boot": {
    "enum": ["yes", "no", "preserve"],
    "description": "Set the first boot behavior of the /etc/machine-id file in the tree"
  }
}
```
instead of "preserve" we also could use "preserve-existing", "skip", "ignore". wdyt?

Reply:
Looks good!

No opinion on what to call `preserve`, all sounds reasonable to me :)

Reply:
I went with ["yes", "no", "preserve"] - thank you!